[2024-12-24T19:29:43.546+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shift_table_sick_leave_dag.load_data_2 manual__2024-12-24T19:24:38.753770+00:00 [queued]>
[2024-12-24T19:29:43.553+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shift_table_sick_leave_dag.load_data_2 manual__2024-12-24T19:24:38.753770+00:00 [queued]>
[2024-12-24T19:29:43.554+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2024-12-24T19:29:43.567+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data_2> on 2024-12-24 19:24:38.753770+00:00
[2024-12-24T19:29:43.571+0000] {standard_task_runner.py:57} INFO - Started process 3070 to run task
[2024-12-24T19:29:43.574+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shift_table_sick_leave_dag', 'load_data_2', 'manual__2024-12-24T19:24:38.753770+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/shift_table_sick_leave_dag.py', '--cfg-path', '/tmp/tmp5gfab546']
[2024-12-24T19:29:43.576+0000] {standard_task_runner.py:85} INFO - Job 200: Subtask load_data_2
[2024-12-24T19:29:43.594+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-24T19:29:43.629+0000] {task_command.py:415} INFO - Running <TaskInstance: shift_table_sick_leave_dag.load_data_2 manual__2024-12-24T19:24:38.753770+00:00 [running]> on host e335f8873093
[2024-12-24T19:29:43.707+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shift_table_sick_leave_dag' AIRFLOW_CTX_TASK_ID='load_data_2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-24T19:24:38.753770+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-24T19:24:38.753770+00:00'
[2024-12-24T19:29:43.847+0000] {base.py:73} INFO - Using connection ID 'admin' for task execution.
[2024-12-24T19:29:43.879+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-24T19:29:43.889+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=shift_table_sick_leave_dag, task_id=load_data_2, execution_date=20241224T192438, start_date=20241224T192943, end_date=20241224T192943
[2024-12-24T19:29:43.908+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-24T19:29:43.924+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
